erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.js

```diff
@@ -68,52 +64,105 @@ erpnext.maintenance.MaintenanceSchedule = frappe.ui.form.Controller.extend({
 					});
 				}, __("Get Items From"));
 		} else if (this.frm.doc.docstatus === 1) {
-			this.frm.add_custom_button(__('Create Maintenance Visit'), function() {
-				frappe.model.open_mapped_doc({
-					method: "erpnext.maintenance.doctype.maintenance_schedule.maintenance_schedule.make_maintenance_visit",
-					source_name: me.frm.doc.name,
-					frm: me.frm
+			var s = me.frm.doc.schedules;
```

Comment:
variable names must describe what the variable holds.
```suggestion
			let schedules = me.frm.doc.schedules;
```